### DISCUSSION TITLE
PicoGK v1.1.0 with installers for macOS and Windows

### DISCUSSION THREAD

**Discussion by LinKayser** [leap71/PicoGK#14] [General]
Dear all,

the latest release of PicoGK now has **installers for macOS and Windows** (and exhaustive documentation on how to get started).

Download at https://github.com/leap71/PicoGK/releases/tag/PicoGK-v1.1.0

Let me know how well it works. Except for the security warning on macOS, it should work smoothly. 

An **example project is also provided** that allows you to get started right away (it includes PicoGK, ShapeKernel, and @josefinelissner 's new Lattice Library).

Happy coding.
Lin

<img width="812" alt="PicoGKDMG" src="https://github.com/leap71/PicoGK/assets/148448288/c731b769-2703-4b84-b75b-11fa8b4b6dcf">

**Comment by mosri** [comment]
Hi @LinKayser @josefinelissner, thank you for your time and efforts on this project, I appreciate it!

How will future versions of the installer be managed on local computers? 
What would be the best way to stay up-to-date with latest updates without having to keep older versions? I imagine new installs will just overwrite/update files directories or add to the files directories when we download and run the new .dmg installer... in which case we simply just discard the old .dmg installer.

**Reply by LinKayser** [reply]
We will keep the PicoGK runtime library compatible, so updating the runtime using the installer should not break things.

As for the PicoGK C# submodule as well as the other LEAP 71 modules, if you are concerned about keeping them up-to-date, we recommend integrating them as submodule into your GitHub project and pulling the repos at convenient times.

I guess we will work out the details as we go along.

Lin

**Reply by mosri** [reply]
Thanks for the response. Where exactly or what is the file directory that holds the PicoGK installation? When I install using the .dmg file, it says it will install on my "Macintosh HD", but I can't seem to find it anywhere.

Also, what if I wanted to uninstall? I don't intend to, but would like to know. 

Thanks

**Reply by LinKayser** [reply]
Here is information on what the PicoGK installer puts on your system.

https://github.com/leap71/PicoGK/blob/main/Documentation/WhatIsInstalled.md

Lin
